Phung_Van_Chung/test01/class.Fresher.php

```diff
@@ -0,0 +1,215 @@
+<?php
+class FresherPHP
+{
+    public $first_name;
+    public $last_name;
+    public $middle_name;
+    public $birth_day;
+    public $phone_number;
+
+    function __construct(array $arr)
+    {
+        $this->setFirstName($arr['first_name']);
+        $this->setLastName($arr['last_name']);
+        $this->setMiddleName($arr['middle_name']);
+        $this->setBirthDay($arr['birth_day']);
+        $this->setPhoneNumber($arr['phone_number']);
+    }
+
+    // Set function
+    public function setFirstName($fn)
+    {
+        $this->first_name = trim($fn);
+    }
+
+    public function setLastName($ln)
+    {
+        $this->last_name = trim($ln);
+    }
+
+    public function setMiddleName($mn)
+    {
+        $this->middle_name = trim($mn);
+    }
+
+    public function setBirthDay($bd)
+    {
+        $this->birth_day = trim($bd);
+    }
+
+    public function setPhoneNumber($pn)
+    {
+        $this->phone_number = trim($pn);
+    }
+
+    // Get function
+    public function getFirstName()
+    {
+        return $this->first_name;
+    }
+
+    public function getLastName()
+    {
+        return $this->last_name;
+    }
+
+    public function getMiddleName()
+    {
+        return $this->middle_name;
+    }
+
+    public function getBirthDay()
+    {
+        return $this->birth_day;
+    }
+
+    public function getPhoneNumber()
+    {
+        return $this->phone_number;
+    }
+
+    // Get fullname of Fresher
+    public function getFullName()
+    {
+        return ucwords($fullName = $this->getLastName() . "&nbsp" . $this->getMiddleName() . "&nbsp" . $this->getFirstName());
+    }
+
+    // Get Age function
+    public function getAge()
+    {
+        $year = substr($this->getBirthDay(), 6);
+        $now = getdate();
+        $age = $now['year'] - $year;
+        return $age;
+    }
+
+    // Get branch function
+    public function getPhoneBranch()
+    {
+        $result = "";
+        if($this->isViettel())
+        {
+            $result .= "Nhà mạng sử dụng: VIETTEL";
+        } elseif ($this->isVinaphone())
+        {
+            $result .= "Nhà mạng sử dụng: VINAPHONE";
+        } elseif ($this->isMobiphone())
+        {
+            $result .= "Nhà mạng sử dụng: MOBIPHONE";
+        } else
+        {
+            $result .= "Số điện thoại nhập vào định dạng không đúng, hoặc nhà mạng khác";
+        }
+        return $result;
+    }
+
+    public function isViettel()
```

Comment:
Duplicate code in isViettel(), isVinaPhone() and isMobilePhone(). Combine those into one and pass only the array of their distinctive numbers.